rfc/0023-kubewarden-stack-telemetry.md

```diff
@@ -0,0 +1,373 @@
+while the `extraFieldInfo` values will be added as
+[fields](https://github.com/longhorn/upgrade-responder/blob/a6f6c7736b7e420b07ae7d813765dac778ebc638/upgraderesponder/service.go#L558).
+
+For example, the JSON payload above would result in a data point with the
+following tags:
+
+- `appVersion`: The Kubewarden version (e.g., v1.28.0).
+- `kubernetesVersion`: The Kubernetes version where the controller is running
+  (e.g., v1.33.0).
+- `instanceUid`: A UID to uniquely identify the controller instance.
+- `city`: City name extracted from the request's source IP.
+- `country`: Country name extracted from the request's source IP.
+- `country_isocode`: Country ISO code extracted from the request's source IP.
+
+The same data point will have the following fields:
+
+- `policyServerCount`: The number of PolicyServers deployed in the cluster.
+- `policiesCount`: The number of policies deployed in the cluster.
+- `value`: A field always set to `1`,
+  [added](https://github.com/longhorn/upgrade-responder/blob/a6f6c7736b7e420b07ae7d813765dac778ebc638/upgraderesponder/service.go#L560)
+  automatically by `upgrade-responder`.
+
+### Understanding Tags and Fields
+
+There are differences from InfluxDB `tags` and `fields` which may be relevant
+to document here as well. Tags are indexed metadata which can be used to group
+and filter data points. While `fields` are the actual metric data.
+
+| Aspect          | Tags                                                                                | Fields                                                                |
+| :-------------- | :---------------------------------------------------------------------------------- | :-------------------------------------------------------------------- |
+| **Purpose**     | Serve as indexed metadata used for filtering and grouping (e.g., `host`, `region`). | Hold the actual measured values (e.g., `temperature`, `cpu_usage`).   |
+| **Indexing**    | Stored in an inverted index; queries on tags are very fast.                         | Not indexed; queries must scan data, which can be slower.             |
+| **Cardinality** | High cardinality (many unique values) can degrade database performance.             | Low impact on performance; many distinct field values are acceptable. |
+| **Data Types**  | Limited to strings.                                                                 | Support numeric types, booleans, and strings.                         |
+
+### Grafana Dashboard
+
+Alongside the `upgrade-responder` and InfluxDB, a Grafana instance with a custom
+dashboard will be configured. The dashboard will use InfluxDB as its data
+source and provide queries to visualize the desired metrics.
+
+### Telemetry Server Infrastructure
+
+The `upgrade-responder`, InfluxDB, and Grafana instances used for Kubewarden
+telemetry will be maintained by the SUSE team. All members of the Kubewarden
+organization on GitHub will have access to the collected data.
```

Comment:
I would like to be specific here, maybe even linking to the [CONTRIBUTOR_LADDER.md](https://github.com/kubewarden/community/blob/main/CONTRIBUTOR_LADDER.md). Should it be only Kubewarden maintainers?

Reply:
Many thanks! resolving.